server/services/store/storetests/helpers.go

```diff
@@ -8,9 +8,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func InsertBlocks(t *testing.T, s store.Store, container store.Container, blocks []model.Block) {
+func InsertBlocks(t *testing.T, s store.Store, container store.Container, blocks []model.Block, userID string) {
 	for _, block := range blocks {
-		err := s.InsertBlock(container, block)
+		err := s.InsertBlock(container, &block, userID)
```

Comment:
Passing a pointer to a loop var can cause unexpected problems.  I see this was done correctly in other places so this was probably just missed.

Reply:
Nice catch! Fixed.